### DISCUSSION TITLE
Receiving error for 150mb video file

### DISCUSSION THREAD

**Discussion by gweizofficial** [git-lfs/git-lfs#5346] [Q&A]
I'm getting some errors for a 150mb mp4 file. Any help?  Here are the commands:

git lfs install
git lfs track "lore-1080.mp4"
git add .gitattributes
git add .
git commit -m "commit"
git push

and I've attached a screenshot of my errors. Thank you in advance!

![WhatsApp Image 2023-04-22 at 1 16 45 AM (1)](https://user-images.githubusercontent.com/99798695/234193910-ffc2d05a-3b38-4de6-a60a-8dfb83ee34d6.jpeg)

**Comment by bk2204** [comment]
Hey,

This is answered in [the FAQ](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-faq.adoc):

> GitHub rejects large files anywhere in the history of your repository, not just in the latest commit.  If you're still seeing this message, then you have some large files somewhere in your history, even if in the latest commits you've moved them to Git LFS.
>
> To fix this, you can use `git lfs migrate import --everything` with an appropriate `--include` argument.  For example, if you wanted to move your `.jpg` and `.png` files into Git LFS, you can do that with `git lfs migrate import --everything --include="*.jpg,*.png"`.  More complicated patterns are possible: run `git help gitattributes` for more information on valid patterns.  Note that if you're specifying directories, using slashes is mandatory: backslashes are not allowed as path separators.

**Reply by bk2204** [reply]
Also note that in your case you've written `lore-1080.mp4`, but your file is `Lore 1080.mp4`, which is not the same.  The pattern has to match the file exactly, including case, if you want it to work.
